### PR TITLE
Add feature commands for debugger python script

### PR DESCRIPTION
**Pull request by polaroi8d**
* add quit command
* add all arguments to delete command, which in the first state delete all active breakpoints from the program
* add test case, for clear and stop command
* change some test files

JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.py [hunk 1]
```diff
@@ -167,10 +167,18 @@ def exec_command(self, args, command_id):
     def do_continue(self, args):
         """ Continue execution """
         self.exec_command(args, JERRY_DEBUGGER_CONTINUE)
+        print("(jerry-debugger) Use \'stop\' command to pause the debugger.")
+        self.cmdloop()
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: Is this a recursive call? My problem is that the network message processing should not be stop when continue is executed.

Comment (polaroi8d) [inline jerry-debugger/jerry-client-ws.py]: Why would you think, the message processing should stop?

Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: Becasue cmdLoop() call is blocking. Or it is not?

**Comment by polaroi8d** [comment]
@zherczeg Updated the PR.

**Comment by polaroi8d** [comment]
@zherczeg Updated the PR.

File: jerry-debugger/jerry-client-ws.py [hunk 2]
```diff
@@ -15,14 +15,15 @@
 # limitations under the License.
 
 from __future__ import print_function
-import socket
-import sys
+from cmd import Cmd
+from pprint import pprint  # For the readable stack printing.
+from struct import *
```
Comment (LaszloLango) [inline jerry-debugger/jerry-client-ws.py]: Please don't use wildcard import.

File: jerry-debugger/jerry-client-ws.py [hunk 3]
```diff
@@ -234,7 +257,7 @@ def exec_backtrace(self, args):
                 print("Error: Positive integer number expected")
                 return
 
-        message = pack(self.debugger.byte_order + "BBIB" + self.debugger.idx_format,
+        message = struct.pack(self.debugger.byte_order + "BBIB" + self.debugger.idx_format,
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: Misaligned lines? Lots of similar changes below.

File: tests/debugger/do_delete_all.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// limitations under the License.
+
+var str = "Delete all breakpoints";
+var patt = new RegExp("e");
+var res = patt.test(str);
+var pi = Math.PI;
+
+function delete_test(x,y) {
+	var sum = x + y;
+}
```
Comment (zherczeg) [inline tests/debugger/do_delete_all.js]: Missing newline.

**Pull request merged**